### ISSUE TITLE
Critical Defect: Remove Sentry usage from this SDK (for real) 

### ISSUE THREAD

**Issue by boennemann** [idnow/de.idnow.ios#81]
It seems like we're back to square one, after waiting for #56 for half a year.

After some very intense debugging sessions we were incredibly "happy" to find out, that you didn't remove Sentry SDK usage, but only the dependency definition from the cocoa spec.

As our app is also using Sentry it seems like the IDnow SDK picks up the existing Sentry SDK and then crashes – probably because it is an incompatible version.

```
EXC_BAD_ACCESS
 858D01CB1FB34C48AFF2E3D22176C110 > setLevel: >
Attempted to dereference garbage pointer 0x3.
```

This happens as soon as we try to capture an exception. Thus we have lost all our own error reports, because this SDK destroyed our error handling.

We're back to version 3.19.0, the last "working" version of this SDK. After downgrading to this version our own Sentry usage is immediately working again.

#56 has mentioned every single reason why an *SDK* should not do any of this, please follow through on the effort of 4.1.0 and remove Sentry, but for real this time.

We are hoping not to spend the next half year pinging you on this issue. An immediate update and reaction is absolutely necessary.

@arminbauer @souheib93 @phu-idnow @hkretzschmar @kbarzen-idn @BilelSelmiIDnow @sleski @SlawomirLeski @vmalinskyi @philipp-idnow

**Comment by tjanela** [comment]
🤯

**Comment by phu-idnow** [comment]
Hi @boennemann  
thanks for pointing out the issue, we will feedback you soon!

**Comment by tjanela** [comment]
@boennemann see #68 for a similar issue.

**Comment by mtidnow** [comment]
We are looking at it and will provide a solution asap. Thanks, M.

**Comment by phu-idnow** [comment]
hi all,
The team already fixed the issue since SDK version 4.1.1, so please update your Podfile 

Thanks,

**Comment by NazarYablonskiy** [comment]
I can confirm that after upgrading from 4.1.6 to 4.1.7, `Sentry` disappeared from Podfile.lock

Good job, IDNow team!
